server/src/api/router.go

```diff
@@ -198,6 +198,8 @@ func (s *Server) initBoardSessionResources(r chi.Router) {
 
 		r.With(s.BoardModeratorContext).Put("/", s.updateBoardSessions)
```

Comment:
This endpoint just quite the same to the newly addition, except `/ready` only changes the ready state and this resource is also capable of e.g. changing the raised hand state for all participants. If it wasn't working before we should probably fix that :)